src/api/controllers/articleController.js

```diff
@@ -41,15 +57,13 @@ export default class ArticleController {
     let articleTags;
 
     const articleValidInput = await articleValidator(req.body);
-    const { category: id } = req.body;
     const readTime = generateReadtime(req.body.body);
+    const { category } = req.body;
```

Comment:
@alainburindi why this change?

Reply:
while creating it! because it is just a draft